conans/test/integration/command/create_test.py

```diff
@@ -370,7 +369,7 @@ def test_create_format_json():
                  ....
                 },
                 {'ref': 'hello/0.1#18d5440ae45afc4c36139a160ac071c7',
-                 'requires': {'2': 'pkg/0.2#db78b8d06a78af5c3ac56706f133098d'},
+                 'dependencies': {'2': 'pkg/0.2#db78b8d06a78af5c3ac56706f133098d'},
```

Comment:
This doesn't look good, the inner thing should be a dict, not a reference, shouldn't it?

Reply:
This is a docstring. That's the reason why the test is passing. Let me update it.